.github/workflows/tests.yml

```diff
@@ -174,30 +174,27 @@ jobs:
 
     - name: Build Docker image
       if: matrix.os == 'ubuntu-latest'
-      run: |
-        docker pull homebrew/brew
-        docker-compose -f Dockerfile.yml build sut
+      run: docker build -t brew .
 
     - name: Run brew test-bot
       run: |
         if [ "$RUNNER_OS" = "Linux" ]; then
-          docker-compose -f Dockerfile.yml run --rm -v $GITHUB_WORKSPACE:/tmp/test-bot sut
-          docker tag homebrew_sut brew
+          docker run --rm -v $GITHUB_WORKSPACE:/tmp/test-bot brew brew test-bot
```

Comment:
@MikeMcQuaid @reitermarkus Why is this `-v` volume mount option here? Does `brew test-bot` produce any build artifacts used later? It doesn't look like it.

Reply:
Probably just copied from a place where the artefacts were used later.

Reply:
I've removed it.